docs/maintain/maintain-guides-how-to-validate-polkadot.md

```diff
@@ -304,9 +303,9 @@ If you run into compile errors, you may have to switch to a less recent nightly.
 done by running:
 
 ```sh
-rustup install nightly-2021-06-09
-rustup target add wasm32-unknown-unknown --toolchain nightly-2021-06-09
-cargo +nightly-2021-06-09 build --release
+rustup install nightly-2022-05-18
+rustup target add wasm32-unknown-unknown --toolchain nightly-2022-05-18
+cargo +nightly-2022-05-18 build --release
```

Comment:
do we always need to specify a specific rust nightly date? they cannot just always try latest nightly?

Reply:
We should say to check the release notes, as the rust versions are pinnned to the release that our clean room machine builds our binaries with that successfully work

Reply:
I updated the Rust compiler recommendations and suggested to check out the release notes for the exact version if one runs into the compilation errors.

Reply:
I think we use new Rust features almost as quickly as they come out so I think it's probably better to do what Shawn is suggesting: go with the latest nightly.

Reply:
Put in in here https://github.com/w3f/polkadot-wiki/issues/3653